serve.sh

```diff
@@ -1 +1 @@
-bundler exec jekyll serve --future
\ No newline at end of file
+bundler exec jekyll serve --future --incremental
```

Comment:
Was this working fine for you?
Incremental mode was buggy when I was using it, list pages would not regenerate sometimes
Notice on the site that they say it's an experimental flag